CHANGELOG.md

```diff
@@ -86,6 +86,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fix segfault when dereferencing `ffi::PyDateTimeAPI` without the GIL. [#1563](https://github.com/PyO3/pyo3/pull/1563)
 - Fix memory leak when converting to u128 and i128. [#1638](https://github.com/PyO3/pyo3/pull/1638)
 - Fix segfault when calling `PyList::get_item` with negative indices. [#1668](https://github.com/PyO3/pyo3/pull/1668)
+- Fix inability to call `Py::as_ref()` against `PySequence`. [#1682](https://github.com/PyO3/pyo3/pull/1682)
```

Comment:
I think is should go to the `### Added` part since it's a new API not a bugfix.

Reply:
That's what I was wondering.  `PySequence` had already implemented `PyTryFrom` at the time I had to take a look into this, so I thought it simply didn't get modified to reflect the past changes to `Py`.

Reply:
Agreed this is an `Added` API.